ddtrace/context.py

```diff
@@ -29,6 +29,21 @@ def __init__(self):
         self._finished_spans = 0
         self._current_span = None
         self._lock = threading.Lock()
+        self._base_parent_span_id = None
+        self._base_parent_trace_id = None
+
+    def set_base_parent_span_ids(self, trace_id, span_id):
```

Comment:
yes it's possible that the `helpers.run_in_executor` doesn't cover all possible cases... for now let's stick with your `Task` linking function, so that later on we can open a new PR to improve other helpers (i.e. if `patch()` is called, also other helpers will wrap the stdlib).

Thank you!